src/components/accounts/AccountFormAccountId.js

```diff
@@ -40,10 +40,16 @@ class AccountFormAccountId extends Component {
         const {
             formLoader,
             requestStatus,
-            autoFocus
-        } = this.props;
+            autoFocus,
+            type
+        } = this.props
 
-        const { accountId } = this.state;
+        const { accountId } = this.state
+
+        const requestStatusSameAccount = type === 'send-money' && this.props.accountId === accountId && {
```

Comment:
@vgrichina If we removed `type` prop for `AccountFormAccountId` component (#339), I will also rebuild this one. This change is only for Send Money page, so maybe just check it by URL instead of `type` prop?

Reply:
@marcinbodnar I think it makes sense to make check / pass `requestStatus` from outside component instead. Isn't it passed as prop already?

Reply:
@vgrichina 
>Isn't it passed as prop already?
It is at #339 changes. I will add it also here and move this check to the top-level component.

Reply:
@marcinbodnar any update on this?

Reply:
@vgrichina planning to finish it today

Reply:
@vgrichina changes ready